src/components/pages/dashboard/dashboard.js

```diff
@@ -66,38 +72,41 @@ export class Dashboard extends Component {
     // Telemetry stream - START
     const onPendingStart = () => this.setState({ telemetryIsPending: true });
 
-    const telemetry$ = TelemetryService.getTelemetryByDeviceIdP15M()
+    const getTelemetryStream = ({ deviceIds = [] }) => TelemetryService.getTelemetryByDeviceIdP15M(deviceIds)
       .merge(
         this.telemetryRefresh$ // Previous request complete
           .delay(Config.telemetryRefreshInterval) // Wait to refresh
           .do(onPendingStart)
-          .flatMap(_ => TelemetryService.getTelemetryByDeviceIdP1M())
+          .flatMap(_ => TelemetryService.getTelemetryByDeviceIdP1M(deviceIds))
       )
       .flatMap(transformTelemetryResponse(() => this.state.telemetry))
       .map(telemetry => ({ telemetry, telemetryIsPending: false })); // Stream emits new state
       // Telemetry stream - END
 
       // KPI stream - START
-      const currentFrom = 'NOW-PT1H';
-      const previousFrom = 'NOW-PT2H';
-
-      const currentParams = { from: currentFrom, to: 'NOW' };
-      const previousParams = { from: previousFrom, to: currentFrom };
 
       // TODO: Add device ids to params - START
-      const kpis$ = this.panelsRefresh$
+      const getKpiStream = ({ deviceIds = [], timeInterval }) => this.panelsRefresh$
```

Comment:
are moving away from $ convention for stream?

Reply:
No, in this case getKpiStream is a function, not a stream anymore. Now it returns a stream.